cpp/cmake/thirdparty/patches/thrust_disable_64bit_dispatching.diff

```diff
@@ -1,29 +1,25 @@
-diff --git a/thrust/system/cuda/detail/dispatch.h b/thrust/system/cuda/detail/dispatch.h
-index d0e3f94..76774b0 100644
---- a/thrust/system/cuda/detail/dispatch.h
-+++ b/thrust/system/cuda/detail/dispatch.h
-@@ -32,9 +32,8 @@
+diff --git a/thrust/thrust/system/cuda/detail/dispatch.h b/thrust/thrust/system/cuda/detail/dispatch.h
+index d0e3f94ec..5c32a9c60 100644
+--- a/thrust/thrust/system/cuda/detail/dispatch.h
++++ b/thrust/thrust/system/cuda/detail/dispatch.h
+@@ -32,8 +32,7 @@
          status = call arguments; \
      } \
      else { \
 -        auto THRUST_PP_CAT2(count, _fixed) = static_cast<thrust::detail::int64_t>(count); \
 -        status = call arguments; \
--    }
-+       throw std::runtime_error("THRUST_INDEX_TYPE_DISPATCH 64-bit count is unsupported in libcudf"); \
-+    }
++        throw std::runtime_error("THRUST_INDEX_TYPE_DISPATCH 64-bit count is unsupported in libcudf"); \
```

Comment:
Not sure if I'm reading this diff right (GH is rendering the diff files rather strangely) but this looks like we have one fewer curly brace than before. Is that right?

Reply:
I don't think there's any significant change to braces here. The previous patch had an extra +/-, perhaps. You can compare them manually:

Before: https://github.com/rapidsai/cudf/blob/branch-24.02/cpp/cmake/thirdparty/patches/thrust_disable_64bit_dispatching.diff

After: https://github.com/bdice/cudf/blob/cccl-2.2.0/cpp/cmake/thirdparty/patches/thrust_disable_64bit_dispatching.diff